Functions.Templates/Templates/EdgeHubTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "defaultFunctionName": "EdgeHubTrigger",
+    "description": "$EdgeHubTrigger_description",
```

Comment:
You need to add a string with the description of the template to the https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Resources/Resources.resx file. Here is a sample string `BlobTrigger_description`

Reply:
Added